smctools_common.h

```diff
@@ -140,6 +140,11 @@ enum {
 	SMC_NLA_LGR_R_CONNS_NUM,	/* u32 */
 	SMC_NLA_LGR_R_V2_COMMON,	/* nest */
 	SMC_NLA_LGR_R_V2,		/* nest */
+	SMC_NLA_LGR_R_NET_COOKIE,       /* u64 */
+	SMC_NLA_LGR_R_PAD,              /* flag */
+	SMC_NLA_LGR_R_BUF_TYPE,         /* u8 */
```

Comment:
They are not used, yet. I'm fine with them being added in order to be complete.
Please use tabs infront of the comments instead of spaces for consistency.

Reply:
Sure, will improve this.